public/react-chat/app.jsx

```diff
@@ -1,5 +1,5 @@
 // A placeholder image if the user does not have one
-const PLACEHOLDER = '/placeholder.png';
+const PLACEHOLDER = 'https://placeimg.com/60/60/people';
```

Comment:
The guides had those links in for a while already which is why I ended up changing it. I can remove the placeholder from the repo.